src/main/java/frc/robot/scoreStates/AmpState.java

```diff
@@ -73,23 +61,11 @@ public Command driveToClosestOptimalPoint() {
 
     @Override
     public Command score() {
-        return Commands.sequence(
-                driveToClosestOptimalPoint(),
-                Commands.runOnce(() -> SwerveDrive.getInstance().lock()),
-                Commands.parallel(
-                        elevator.setHeight(CommandGroupsConstants.MAX_HEIGHT),
-                        Commands.either(
-                                gripper.setRollerAndWrist(
-                                        GripperConstants.AMP_POWER_NORMAL,
-                                        CommandGroupsConstants.WRIST_ANGLE_AMP_FORWARD),
-                                gripper.setRollerAndWrist(
-                                        GripperConstants.AMP_POWER_REVERSE,
-                                        CommandGroupsConstants.WRIST_ANGLE_AMP_BACKWARDS),
-                                () -> isAmpingForward)));
+        return null;
     }
 
     @Override
     public Command finalizeScore() {
-        return CommandGroups.getInstance().retractGrillevator();
+        return ScoreState.super.finalizeScore();
     }
 }
```

Comment:
`ScoreState`'s `finalizeScore()` has a default implementation:
https://github.com/Galaxia5987/robot-2024/blob/830d7254d5fcff26014ba8d08b2f755258828da2/src/main/java/frc/robot/scoreStates/ScoreState.java#L23-L25
You can just delete the implementation here.

Reply:
b6110bb35c20abf310b1f31b2097ec84e007a7ad